app/models/npq_application.rb

```diff
@@ -110,8 +110,8 @@ def in_uk_catchment_area?
     UK_CATCHMENT_AREA.include?(teacher_catchment)
   end
 
-  def self.participant_declaration_finder(participant_identity_id)
-    ParticipantDeclaration::NPQ.find_by_participant_profile_id(ParticipantProfile.find_by_participant_identity_id(participant_identity_id)&.id)
+  def self.completed_participant_declaration_finder(participant_identity_id)
+    ParticipantDeclaration::NPQ.find_by(declaration_type: "completed", participant_profile_id: ParticipantProfile.find_by_participant_identity_id(participant_identity_id)&.id)
```

Comment:
we can have multiple completed declarations, one could be voided and then another one could be the correct one. Maybe we should select the billable one?